modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/line-chart/index.js

```diff
@@ -71,7 +72,9 @@ const LineChart = ({
 
 								const value = data[0].value.toFixed(1);
 
-								return `<div class="line-chart-tooltip w-100 bg-neutral-0 d-flex font-weight-bold rounded-sm p-2"><span class="d-flex font-weight-normal mr-2 w-100">${title}</span> $${value}</div>`;
+								return hasPersonalizedTooltip
+									? `<div class="line-chart-tooltip w-100 bg-neutral-0 d-flex font-weight-bold rounded-sm p-2"><span class="d-flex font-weight-normal mr-2 w-100">${title}</span> ${value}</div>`
+									: `<div class="line-chart-tooltip w-100 bg-neutral-0 d-flex font-weight-bold rounded-sm p-2"><span class="d-flex font-weight-normal mr-2 w-100">${title}</span> $${value}</div>`;
```

Comment:
This custom tooltip has the same structure. Please make it as a function that returns this structure to your chart.

Reply:
Done